src/App.css

```diff
@@ -217,35 +217,36 @@ h2 {
   outline: none; 
 }
 
-/* login */
+/* login , register pages */
 
-.login-container {
+.login-container, .register-container {
   max-width: 400px;
   margin: 50px auto;
   padding: 20px;
   border: 1px solid #ccc;
   border-radius: 8px;
   background-color: #f9f9f9;
-  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+  box-shadow: 0 2px 4px #0000001a;
   display: flex;
   flex-direction: column;
   align-items: center;
 }
 
-.login-container h1 {
+.login-container h1, .register-container h1 {
```

Comment:
Same for all other classes that are changes. YOu're trying to do consistent design but repeat the code all the time. Instead, use shared components and only add styles (and classes) once, then reuse component. That's basically why we use react in fist place.